### DISCUSSION TITLE
Direct database access

### DISCUSSION THREAD

**Discussion by jesperordrup** [pocketbase/pocketbase#5393] [Q&A] [answered]
Is it possible to access and write to db directly in the server side functions /events?

* to make performant mass updates

I know theres a refactor process but it would still be good to know if db is accessable direct

**Comment by ganigeorgiev** [comment, marked as answer]
Please refer to the ["Use as framework" section](https://pocketbase.io/docs/use-as-framework/) and more specifically:

- Go:
  - https://pocketbase.io/docs/go-database/
  - https://pocketbase.io/docs/go-records/
 
- JS `pb_hooks`:
  - https://pocketbase.io/docs/js-database/
  - https://pocketbase.io/docs/js-records/

**Reply by ganigeorgiev** [reply]
If you are asking for direct DB access over the Web APIs using the SDKs - then No and even if we eventually allow running raw SQL queries over the Web APIs (https://github.com/pocketbase/pocketbase/issues/2236) this functionality will be limited only for admins.

**Reply by jesperordrup** [reply]
> If you are asking for direct DB access over the Web APIs using the SDKs - t

No :-) I'm thinking pb_hooks. I just have misunderstood the message on https://pocketbase.io/docs/js-records/#create-new-record) as I didnt think it was more direct.

Thank you
